internal/service/common/utils/utils.go

```diff
@@ -174,3 +174,21 @@ func CompareObjects[T db.Model](a, b T, excluded ...string) DBTag {
 
 	return tags
 }
+
+// GetColumns gets the DB column names for the given fields
+func GetColumns[T db.Model](record T, fields []string) []string {
+	tags := GetDBTagsFromStructFields(record, fields...)
+	columns := make([]string, len(fields))
+	for i, field := range fields {
+		columns[i] = tags[field]
+	}
+	return columns
+}
+
+func GetColumnsAsAny(slice []string) []any {
```

Comment:
what we do is to call GetDBTagsFromStructFields with the fields you want and then .Columns

Reply:
Using slice instead to assert order